jupyter_book/yaml.py

```diff
@@ -90,6 +90,8 @@ def yaml_to_sphinx(yaml, config):
         sphinx_config["jupyter_execute_notebooks"] = execute.get("execute_notebooks")
```

Comment:
just a note that this is here well, it's checking for the execute config [from this page](https://jupyterbook.org/content/execute.html#trigger-notebook-execution), so check which one to use (again, apologies for the slightly confusing config setup)

Reply:
Thanks for the additional info - I need to spend a bit more time mapping out the different objects/containers that hold configuration information in the different layers (jb, myst-nb, etc.) and how they map to each other.